server/controllers/donations.js

```diff
@@ -61,11 +43,11 @@ const stripeDonation = (req, res, next) => {
     return next(new errors.BadRequest('Stripe Token missing.'));
   }
 
-  if (!amountFloat) {
+  if (!amount) {
     return next(new errors.BadRequest('Payment Amount missing.'));
   }
 
-  if (amountInt < 50) {
+  if (amount < 50) {
```

Comment:
As the comment on our API suggests, people by default will assume that "amount" is in float, not in cents. That's why I'd rather be explicit. The more we can avoid "RTFM", the better.